Extensions/TileMap/tilemapruntimeobject-pixi-renderer.js

```diff
@@ -43,6 +35,11 @@ gdjs.TileMapRuntimeObjectPixiRenderer.prototype.getRendererObject = function() {
   return this._pixiObject;
 };
 
+gdjs.TileMapRuntimeObjectPixiRenderer.prototype.incrementAnimationFrameX = function(runtimeScene){
+  console.log("update")
+  this._pixiRenderer.plugins.tilemap.tileAnim[0] += 1;
```

Comment:
This is a global variable - so this means that if I create two different tilemaps, they will be animated at the same time. Is this normal? Could we avoid this somehow?

Reply:
I'm not sure if we can :( pixi-tilemap gets lumped in pixi's plugins once
initiated, after which it's api expects to be animated by incrementing the
global value :( Perhaps a question for @ivan